src/Tests/Integration/fixtures/banks/visaCal/02-pre-login/frame-7.html

```
@@ -0,0 +1,4 @@
+<html>
```

HTMLHint (1.9.2)

[warning] An lang attribute must be present on <html> elements.

(html-lang-require)